src/rules/prefer-params-over-parent-id.ts

```
@@ -198,12 +250,12 @@ export const preferParamsOverParentId = createRule<[], MessageIds>({
               data: {
                 paramName: paramSuggestion,
               },
-              fix(fixer) {
+              fix: paramsInScope ? (fixer) => {
```

ESLint

[error] Insert `⏎···············`

(prettier/prettier)